.github/workflows/update-bidi-types.yml

```diff
@@ -31,24 +31,32 @@ jobs:
     steps:
       - name: Check out repository
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - name: Set up Python
+        uses: actions/setup-python@bd6b4b6205c4dbad673328db7b31b7fab9e241c0 # v4.6.1
```

Comment:
I understand you added python because of `npm run format` which runs `pre-commit` which runs `yapf`. But ultimately this workflow doesn't care about python at all. Adding this dependency here feels wrong.

There are multiple ways to fix this. If you're not keen on fixing it now then please add a comment above:

```
XXX: Remove python dependency (used indirectly by `yapf` via `npm run format`.
```

Reply:
`pre-commit` also comes directly from Python.